Scripts/import-logs.ps1

```diff
@@ -0,0 +1,110 @@
+#########powershell script to upload log files into s3 bucket###########
+
+#s3Bucket path to save the log files
+param (
+    [string] $version,
+    [string] $ReleaseId
+)
+$Lansaversion = $version
```

Comment:
I can see no value in having local copies of the parameters. Please rename the parameters to be the Local names and remove these 2 lines.
As in
```
param (
    [string] $Lansaversion,
    [string] $PipelineReleaseId 
)
```

You also need to receive the List of Database Types that are being tested and only pick up THOSE log files.